base/src/main/java/io/spine/validate/ByteStringFieldValidator.java

```diff
@@ -57,11 +55,14 @@ protected boolean isValueNotSet(ByteString value) {
         return result;
     }
 
+    /**
+     * {@inheritDoc}
+     *
+     * <p>Performs no validation.
+     */
     @Override
-    protected List<ConstraintViolation> validate() {
-        checkIfRequiredAndNotSet();
-        final List<ConstraintViolation> violations = super.validate();
-        return violations;
+    protected void doValidate() {
+        // NoOp
```

Comment:
Requires some explanation in JavaDocs. I am not sure that's correct, as there may be some rules applied to it. Not right now, but in general — it's always a good idea to ask `super`.

Same below.